src/js/core.js

```diff
@@ -643,6 +596,54 @@ function MediumEditor(elements, options) {
         },
 
         /**
+         * NOT DOCUMENTED - exposed as a helper for other extensions to use
+         */
+        addBuiltInExtension: function (name) {
+            var extension = this.getExtensionByName(name);
+            if (extension) {
+                return extension;
+            }
+
+            switch(name) {
+                case 'anchor':
+                    extension = new MediumEditor.extensions.anchor(this.options.anchor);
+                    break;
+                case 'anchorPreview':
+                    extension = new MediumEditor.extensions.anchorPreview(this.options.anchorPreview);
+                    break;
+                case 'autoLink':
+                    extension = new MediumEditor.extensions.autoLink();
+                    break;
+                case 'fontsize':
+                    extension = new MediumEditor.extensions.fontSize();
+                    break;
+                case 'imageDragging':
+                    extension = new MediumEditor.extensions.imageDragging();
+                    break;
+                case 'keyboardCommands':
+                    extension = new MediumEditor.extensions.keyboardCommands(this.options.keyboardCommands);
+                    break;
+                case 'paste':
+                    extension = new MediumEditor.extensions.paste(this.options.paste);
+                    break;
+                case 'placeholder':
+                    extension = new MediumEditor.extensions.placeholder(this.options.placeholder);
+                    break;
+                default:
+                    if (ButtonsData.hasOwnProperty(name)) {
```

Comment:
Could you add a short comment saying that every buttons (*bold*, *italic*, etc..) is loaded as an extension?
I says that because it wasn't clear for me at first read